### DISCUSSION TITLE
Blocks required to make consoles?

### DISCUSSION THREAD

**Discussion by Chew** [eccentricdevotion/TARDIS#576] [Wiki]
I was checking the [TARDIS console seed block types](https://eccentricdevotion.github.io/TARDIS/creating-a-tardis.html#tardis-types) and noticed it was a bit inaccurate? The division console isn't there, and some galleries are missing.

I can probably get them all added and updated if I know the block for these. Maybe also have a link in each name to each interior, if they all exist. Will go through and see which ones are missing, but I won't be able to take the images myself (I don't think).

There's 28 known interiors, is there a block for all of these? If it's in the code I can grab them.

**Comment by eccentricdevotion** [comment]
From the [console loader](https://github.com/eccentricdevotion/TARDIS/blob/master/src/main/java/me/eccentric_nz/TARDIS/files/TARDISConsoleLoader.java):
```
ARS -> QUARTZ_BLOCK
BIGGER -> GOLD_BLOCK
BUDGET -> IRON_BLOCK
CAVE -> DRIPSTONE_BLOCK
COPPER -> WARPED_PLANKS // designed by vistaero
CORAL -> NETHER_WART_BLOCK // designed by vistaero
DELTA -> CRYING_OBSIDIAN
DELUXE -> DIAMOND_BLOCK
DIVISION -> PINK_GLAZED_TERRACOTTA
ELEVENTH -> EMERALD_BLOCK, 
ENDER -> PURPUR_BLOCK // designed by ToppanaFIN
FACTORY -> YELLOW_CONCRETE_POWDER // designed by Razihel
MASTER -> NETHER_BRICKS // designed by ShadowAssociate
PLANK -> BOOKSHELF
PYRAMID -> SANDSTONE_STAIRS // designed by airomis
REDSTONE -> REDSTONE_BLOCK
ROTOR -> HONEYCOMB_BLOCK
STEAMPUNK -> COAL_BLOCK
THIRTEENTH -> ORANGE_CONCRETE // designed by Razihel
TOM -> LAPIS_BLOCK
TWELFTH -> PRISMARINE
WAR -> WHITE_TERRACOTTA
WEATHERED -> WEATHERED_COPPER
LEGACY_BIGGER -> ORANGE_GLAZED_TERRACOTTA
LEGACY_BUDGET -> LIGHT_GRAY_GLAZED_TERRACOTTA
LEGACY_DELUXE -> LIME_GLAZED_TERRACOTTA
LEGACY_ELEVENTH -> CYAN_GLAZED_TERRACOTTA
LEGACY_REDSTONE -> RED_GLAZED_TERRACOTTA
```
Will grab some screenshots

**Comment by eccentricdevotion** [comment]
CAVE
![CAVE](https://eccentricdevotion.github.io/TARDIS/images/consoles/cave.jpg)
COPPER
![COPPER](https://eccentricdevotion.github.io/TARDIS/images/consoles/copper.jpg)
DELTA
![DELTA](https://eccentricdevotion.github.io/TARDIS/images/consoles/delta.jpg)
DIVISION
![DIVISION](https://eccentricdevotion.github.io/TARDIS/images/consoles/division.jpg)
FACTORY
![FACTORY](https://eccentricdevotion.github.io/TARDIS/images/consoles/factory.jpg)
ROTOR
![ROTOR](https://eccentricdevotion.github.io/TARDIS/images/consoles/rotor.jpg)
THIRTEENTH
![THIRTEENTH](https://eccentricdevotion.github.io/TARDIS/images/consoles/thirteenth.jpg)
WAR
![WAR](https://eccentricdevotion.github.io/TARDIS/images/consoles/war.jpg)
WEATHERED
![WEATHERED](https://eccentricdevotion.github.io/TARDIS/images/consoles/weathered.jpg)

**Reply by Chew** [reply]
As soon as I tab in, this pops up 😄 

Which ones are which?

**Reply by eccentricdevotion** [reply]
There's also:
Division Vanilla
![DIVISION](https://eccentricdevotion.github.io/TARDIS/images/consoles/division_vanilla.jpg)
Thirteenth Vanilla
![DIVISION](https://eccentricdevotion.github.io/TARDIS/images/consoles/thirteenth_vanilla.jpg)
You can choose...

**Comment by eccentricdevotion** [comment]
Labelled

**Comment by eccentricdevotion** [comment]
Images have been pushed to gh-pages branch
https://github.com/eccentricdevotion/TARDIS/tree/gh-pages/images/consoles

**Comment by Chew** [comment]
Made #577 

I'm heading to sleep since it is now 1 AM, so will check any reviews tomorrow...
